config-cli/src/test/resources/keygen-sample.json

```diff
@@ -5,10 +5,53 @@
         "password": "tiger",
         "url": "foo:bar"
     },
-    "server": {
-        "hostName": "http://localhost",
-        "port": 99
-    },
+    "serverConfigs":[
+        {
+            "app":"ThirdParty",
+            "enabled": true,
+            "serverSocket":{
+                "type":"INET",
+                "port": 8090,
+                "hostName": "http://localhost"
+            },
+            "communicationType" : "REST"
+        },
+        {
+            "app":"Q2T",
+            "enabled": true,
+            "serverSocket":{
+                "type":"UNIX",
+                "path":"/tmp/test.ipc"
```

Comment:
Since the socket file is defined here now, is the definition for 'unixSocketFile' still needed (defined further below)?

Reply:
Yes, at least for the time being, for backwards compatibility.